algorithms/backtrack/__init__.py

```diff
@@ -0,0 +1,7 @@
+from .anagram import *
+from .array_sum_combinations import *
+from .combination_sum import *
+from .find_words import *
+from .pattern_match import *
+from .add_operators import *
+from .permute import *
```

Comment:
Why did you skip some files here?

Reply:
unincluded ones were the ones that were not tested.
But I included them now. Ill add more tests later